lib/compiler-llvm/src/translator/intrinsics.rs

```diff
@@ -460,6 +465,145 @@ impl<'ctx> Intrinsics<'ctx> {
+                "wasmer_table_size",
+                i32_ty.fn_type(&[ctx_ptr_ty.as_basic_type_enum(), i32_ty_basic], false),
+                None,
+            ),
+            imported_table_size: module.add_function(
+                "wasmer_imported_table_size",
+                i32_ty.fn_type(&[ctx_ptr_ty.as_basic_type_enum(), i32_ty_basic], false),
+                None,
+            ),
+            table_get: module.add_function(
+                "wasmer_table_get",
+                funcref_ty.fn_type(
+                    &[ctx_ptr_ty.as_basic_type_enum(), i32_ty_basic, i32_ty_basic],
+                    false,
+                ),
+                None,
+            ),
+            imported_table_get: module.add_function(
+                "wasmer_imported_table_get",
+                i8_ptr_ty.fn_type(
+                    &[ctx_ptr_ty.as_basic_type_enum(), i32_ty_basic, i32_ty_basic],
+                    false,
+                ),
+                None,
+            ),
+            table_set: module.add_function(
+                "wasmer_table_set",
+                void_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        funcref_ty.as_basic_type_enum(),
+                    ],
+                    false,
+                ),
+                None,
+            ),
+            imported_table_set: module.add_function(
+                "wasmer_imported_table_set",
+                void_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        funcref_ty.as_basic_type_enum(),
+                    ],
+                    false,
+                ),
+                None,
+            ),
+            table_grow: module.add_function(
+                "wasmer_table_grow",
+                i32_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        funcref_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                    ],
+                    false,
+                ),
+                None,
+            ),
+            imported_table_grow: module.add_function(
+                "wasmer_imported_table_grow",
+                i32_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        funcref_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                    ],
+                    false,
+                ),
+                None,
+            ),
+            func_ref: module.add_function(
+                "wasmer_func_ref",
+                funcref_ty.fn_type(&[i8_ptr_ty_basic, i32_ty_basic], false),
```

Comment:
Does this not take a context pointer? I thought they all did because that's what the backends did by default

Reply:
Yep that's a bug, thanks!